pkg/virt-controller/watch/vmi/lifecycle.go

```diff
@@ -428,6 +429,8 @@ func (c *Controller) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8sv1
 			c.syncVolumesUpdate(vmiCopy)
 		}
 
+		c.syncAutoMigration(vmiCopy)
```

Comment:
@orelmisan can we do something about this name? I think it should contain the word "Condition" but also isn't it networking related?

Reply:
How about `syncMigrationRequiredCondition`?

Currently, SIG network is its only customer, but it could be used to serve other SIGs by replacing the SIG-network migration evaluator with an evaluator that can combine the results of multiple SIGs/flows.

Reply:
Thanks, yes, that would be fine.